pandas/core/window.py

```diff
@@ -780,8 +780,41 @@ def calc(x):
 
 class _Rolling_and_Expanding(_Rolling):
 
-    _shared_docs['count'] = """%(name)s count of number of non-NaN
-    observations inside provided window."""
+    _shared_docs['count'] = dedent(r"""
+    The %(name)s sum if it is a non-Nan value inside the window.
+
+    Returns
+    -------
+    Returns the object type with the sum of the window values
+    in the current row.
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s : Calling object with DataFrames
```

Comment:
Can you link to `Series` / `DataFrame` sum methods?

Reply:
You can take a look at how the other PRs did this, see eg https://github.com/pandas-dev/pandas/pull/19999/files

Reply:
Sounds good, I will update that information.